cfme/configure/tasks.py

```diff
@@ -173,6 +174,11 @@ def all_other_tasks(**filter_kwargs):
     return _get_tasks("tasks_all_other", **filter_kwargs)
 
 
+def delete_all_tasks():
+    toolbar.select('Delete Tasks', 'Delete All', invokes_alert=True)
```

Comment:
This needs to include navigation if we are not yet on the right page (`force_navigate`).